app/src/main/java/ch/admin/bag/dp3t/storage/SecureStorage.java

```diff
@@ -338,21 +338,6 @@ public VaccinationBookingInfoModel getVaccinationBookingInfo(String language) {
 		return map.get(language);
 	}
 
-	public void setVaccinationBookingCantons(VaccinationBookingCantonCollection vaccinationBookingCantonCollection) {
-		prefs.edit().putString(KEY_VACCINATION_CANTONS, gson.toJson(vaccinationBookingCantonCollection)).apply();
```

Comment:
KEY_VACCINATION_CANTONS could be removed entirely. Or did you keep it on purpose, so that it will never be reused in the future?